backend/btrixcloud/orgs.py

```diff
@@ -276,6 +276,13 @@ async def get_org_for_user_by_id(
 
         return Organization.from_dict(res)
 
+    async def get_users_for_org(self, org: Organization, min_role=UserRole.VIEWER):
```

Comment:
```suggestion
    async def get_users_for_org(
        self, org: Organization, min_role=UserRole.VIEWER
    ) -> List[User]:
````
Add return type annotation